src/tello/tello.py

```diff
@@ -488,3 +488,57 @@ def move_up(self, distance) -> str:
         """
 
         return self.move('up', distance)
+
+    def go_xyz_speed(self, x: int, y: int, z: int, speed: int):
+        """Fly to x y z relative to the current position.
+        Speed defines the traveling speed in cm/s.
+        'x', 'y' and 'z' values can't be set between -20 - 20 simultaneously
+        Arguments:
+            x: -500 - 500
+            y: -500 - 500
+            z: -500 - 500
+            speed: 10 - 100
+
+        """
+
+        min_movement = 20
+        max_movement = 500
+        min_speed = 10
+        max_speed = 100
+
+        if x > 0:
+            x = self._check_limits(x, min_movement, max_movement)
+        else:
+            x = self._check_limits(x, -max_movement, -min_movement)
+
+        if y > 0:
+            y = self._check_limits(y, min_movement, max_movement)
+        else:
+            y = self._check_limits(y, -max_movement, -min_movement)
+
+        if z > 0:
+            z = self._check_limits(z, min_movement, max_movement)
+        else:
+            z = self._check_limits(z, -max_movement, -min_movement)
+
+        speed = self._check_limits(speed, min_speed, max_speed)
+
+        cmd = f'go {x} {y} {z} {speed}'
+        self.send_control_command(cmd)
+
+    def _check_limits(self, x: int, min_lim: int, max_lim: int) -> int:
```

Comment:
This function doesn't have to be part of the class, as it doesn't use any of the class variable